### PR TITLE
[Form] Add RadioGroup

### PR DESCRIPTION
**Pull request by MarcoPOtard**

### CODE REVIEW

**Comment by MatouTechnologies** [comment]
This pull request has been deployed to Vercel.
<table>
  <tr>
    <td><strong>✅ Preview:</strong></td>
    <td><a href='https://matou-technologies-purrfect-ui-storybook-18.vercel.app'>https://matou-technologies-purrfect-ui-storybook-18.vercel.app</a></td>
  </tr>
  <tr>
    <td><strong>🔍 Inspect:</strong></td>
    <td><a href='https://vercel.com/matou-technologies/purrfect-ui/7KLCic2y25LLJGcMijVXv59tomFs'>https://vercel.com/matou-technologies/purrfect-ui/7KLCic2y25LLJGcMijVXv59tomFs</a></td>
  </tr>
</table>


<hidden purpose="for-rewritable-pr-comment-action-use" value="deploy-preview-storybook"></hidden>

File: src/form/RadioGroup/RadioGroup.tsx [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+
+  return (
+    <StyledRadioGroup {...props}>
+      <StyledLabel>
+        {props.label}
+        {props.isRequired && <StyledIsRequired> *</StyledIsRequired>}
+      </StyledLabel>
+      {props.radio.map((oneRadio, index) => (
+        <Radio key={index} {...oneRadio} />
+      ))}
```
Comment (Balkeo) [inline src/form/RadioGroup/RadioGroup.tsx:71]:
```suggestion
      {children}
```

File: src/form/RadioGroup/RadioGroup.tsx [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+const StyledFieldError = styled(FieldError)`
+  ${body1};
+  color: ${theme.color.danger};
+`;
+
+type RadioGroupProps = {
+  helperText?: string;
+  label?: string;
+  radio: RadioProps[];
```
Comment (Balkeo) [inline src/form/RadioGroup/RadioGroup.tsx:36]:
```suggestion
  children: React.ReactNode;
```

**Review by Balkeo — CHANGES_REQUESTED** [review]

File: src/form/RadioGroup/RadioGroup.tsx (hunk 1 above)
Comment (MarcoPOtard) [reply src/form/RadioGroup/RadioGroup.tsx:71]: Bien vu et plus logique, modifié

File: src/form/RadioGroup/RadioGroup.tsx (hunk 2 above)
Comment (MarcoPOtard) [reply src/form/RadioGroup/RadioGroup.tsx:36]: Bien vu et plus logique, modifié

File: src/form/Radio/Radio.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,59 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import React from "react";
+import { Label, RadioGroup } from "react-aria-components";
+import styled from "styled-components";
+import { body1, heading1, sizing } from "../../guidelines/theme";
+import Radio from "./Radio";
```
Comment (Balkeo) [inline src/form/Radio/Radio.stories.tsx:6]: Please use absolute Imports

File: src/form/RadioGroup/RadioGroup.spec.tsx [hunk 4]
```diff
@@ -0,0 +1,78 @@
+import { render, screen } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
+import React from "react";
+import { describe, expect, it } from "vitest";
+import Radio from "../Radio/Radio";
+import RadioGroup from "./RadioGroup";
```
Comment (Balkeo) [inline src/form/RadioGroup/RadioGroup.spec.tsx:6]: Please use absolute Imports

File: src/form/RadioGroup/RadioGroup.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,200 @@
+import type { Meta, StoryObj } from "@storybook/react";
+import { fn } from "@storybook/test";
+import React from "react";
+import styled from "styled-components";
+import { heading1, sizing } from "../../guidelines/theme/index";
```
Comment (Balkeo) [inline src/form/RadioGroup/RadioGroup.stories.tsx:5]: Please use absolute Imports

File: src/form/RadioGroup/RadioGroup.tsx [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
+import {
+  RadioGroup as AriaRadioGroup,
+  RadioGroupProps as AriaRadioGroupProps,
+  FieldError,
+  Label,
+  Text,
+} from "react-aria-components";
+import styled from "styled-components";
+import { body1, sizing, theme } from "../../guidelines/theme";
+import { typographies } from "../../guidelines/theme/typographies";
```
Comment (Balkeo) [inline src/form/RadioGroup/RadioGroup.tsx:11]: Please use absolute Imports

File: src/form/Radio/Radio.tsx [hunk 7]
```diff
@@ -0,0 +1,75 @@
+import React from "react";
+import {
+  Radio as AriaRadio,
+  RadioProps as AriaRadioProps,
+} from "react-aria-components";
+import styled from "styled-components";
+import { body1, sizing, theme } from "../../guidelines/theme";
```
Comment (Balkeo) [inline src/form/Radio/Radio.tsx:7]: Please use absolute Imports

File: src/form/Radio/Radio.spec.tsx [hunk 8]
```diff
@@ -0,0 +1,28 @@
+import { render, screen } from "@testing-library/react";
+import React from "react";
+import { describe, expect, it } from "vitest";
+import Radio from "../Radio/Radio";
+import RadioGroup from "../RadioGroup/RadioGroup";
```
Comment (Balkeo) [inline src/form/Radio/Radio.spec.tsx:5]: Please use absolute Imports

**Pull request merged**